addons/mail/tests/test_discuss_controller.py

```diff
@@ -342,3 +343,24 @@ def test_mail_partner_from_email_authenticated(self):
             self.env["res.partner"].search_count([('email', '=', "john2@test.be")]),
             "'mail/message/post' does not create another user if there's already a user with matching email",
         )
+
+    def test_mail_cache_control_header(self):
+        demo = self.authenticate("demo", "demo")
```

Comment:
(new) tests should be made independent of demo data, a specific user should be created here instead